src/components/layout/nav/search/Search.scss

```diff
@@ -0,0 +1,181 @@
+@import "../../../common/style.scss";
+
```

Comment:
i don't think this should be in search.scss, it affects all elements. it should be in app or index or something like that